.github/workflows/test.yml

```diff
@@ -24,9 +23,9 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Set up Python
-        uses: actions/setup-python@v4
+        uses: actions/setup-python@v5
         with:
-          python-version: '3.12'
+          python-version: '3.12.7'
```

Comment:
why we can't go with something like 3.12.* like we do have in others repo even in frontend repos

Reply:
Changes it to 3.12.*